Snakefile

```diff
@@ -360,7 +360,7 @@ rule run_irap:
             echo "irap_single_lib SE finished for {wildcards.sample}"
         else
             # fastq is PE
-            reformat.sh ow=t int=t vpair=t vint=t in={params.root_dir}/{input.fastq} out1=$workingDir/${{localFastqPath}}_1.fastq out2=$workingDir/${{localFastqPath}}_2.fastq
+            reformat.sh ow=t ibq=t int=t vpair=t vint=t in={params.root_dir}/{input.fastq} out1=$workingDir/${{localFastqPath}}_1.fastq out2=$workingDir/${{localFastqPath}}_2.fastq
```

Comment:
Something for future:

`vint` sets `vpair` to true and `interleaved` or `int` to true. so we dont need `int=t vpair=t vint=t`

Reply:
https://github.com/BioInfoTools/BBMap/blob/a9ceda047a7c918dc090de0fdbf6f924292d4a1f/sh/reformat.sh#L41